### PR TITLE
Added db context & migrations

### PR DESCRIPTION
**Pull request by Leeyaoi**

### CODE REVIEW

File: InnoViber.DAL/Message.cs [hunk 1]
```diff
@@ -1,14 +1,10 @@
...
-        public string Text { get; set; } = string.Empty;
-        public int Status { get; set; }
-        public User? User { get; set; }
-        public Chat? Chat { get; set; }
-    }
+    public DateTime Date {  get; set; }
+    public string Text { get; set; } = string.Empty;
+    public int Status { get; set; }
+    public User? User { get; set; }
+    public Chat? Chat { get; set; }
```
Comment (moltenessence) [inline InnoViber.DAL/Message.cs]: Ids are missed: UserId, ChatId

File: InnoViber/Program.cs [hunk 2]
```diff
@@ -9,6 +12,12 @@ public static void Main(string[] args)
             // Add services to the container.
             builder.Services.AddControllersWithViews();
 
+            builder.Services.AddDbContext<ViberContext>(option =>
+            {
+                //option.UseNpgsql("host=localhost;port=5432;dbname=viberDB;user=postgres;password=26088062;connect_timeout=10;sslmode=prefer");
+                option.UseNpgsql("Host=localhost;Port=5432;Database=viberDB;Username=postgres;Password=26088062");
+            });
```
Comment (moltenessence) [inline InnoViber/Program.cs]: why hardcoded here?

**Review by moltenessence — CHANGES_REQUESTED** [review]

File: InnoViber/Program.cs (hunk 2 above)
Comment (pavelGagarin0870) [reply InnoViber/Program.cs]: Delete commented code, retrieve connection string from AppSettings

File: InnoViber/Program.cs [hunk 3]
```diff
@@ -9,6 +12,12 @@ public static void Main(string[] args)
             // Add services to the container.
             builder.Services.AddControllersWithViews();
 
+            builder.Services.AddDbContext<ViberContext>(option =>
```
Comment (pavelGagarin0870) [inline InnoViber/Program.cs]: Move this part of DI to DAL layer

**Review by pavelGagarin0870 — CHANGES_REQUESTED** [review]

**Review by moltenessence — APPROVED** [review]

**Review by pavelGagarin0870 — APPROVED** [review]

**Pull request merged**